### ISSUE TITLE
Getting 409 when creating a new RoleBinding

### ISSUE THREAD

**Issue by vsevel** [fabric8io/kubernetes-client#3188]
This issue has been found in `5.4.0`, which is used in Quarkus `2.0.0.CR1`. And it is a regression compared to `5.3.1`, which was used in Quarkus `2.0.0.Alpha3`.

When running this test:

```
@EnableRuleMigrationSupport
class MyTest {

    @Rule
    public OpenShiftServer server = new OpenShiftServer(true, true);

    @Test
    void mytest() {
        OpenShiftClient client = server.getOpenshiftClient().inNamespace(null);
        Map<String, String> rolebindingLabels = Map.of("mylabel", "myvalue");
        client.rbac().roleBindings().inNamespace("test").create(new RoleBindingBuilder().withNewMetadata().withName("deleteme").withNamespace("test").addToLabels(rolebindingLabels).endMetadata().build());
    }
}
```

In `5.3.1` This is working fine.

In `5.4.0`, it is failing with:

```
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: POST at: https://kubernetes.docker.internal:54931/apis/rbac.authorization.k8s.io/v1/namespaces/test/rolebindings.

	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.requestFailure(OperationSupport.java:639)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.assertResponseCode(OperationSupport.java:578)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:543)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:504)
	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleCreate(OperationSupport.java:292)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.handleCreate(BaseOperation.java:893)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:372)
	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.create(BaseOperation.java:86)
	at com.x.arte.ocpdeploy.deploy.MyTest.mytest(MyTest.java:29)
```

In `5.4.0` I see that the `POST /apis/rbac.authorization.k8s.io/v1/namespaces/test/rolebindings HTTP/1.1` request is played twice. and the `409` happens on the 2nd try, although the response of the first `POST` is a `200`. So I don't understand why the request is replayed.
In `5.3.1`, the request is played only once.

/cc @matthyx

**Comment by geoand** [comment]
Would it be possible to have a release with this fix that we could have in Quarkus the few days so it can be in `2.0.0.Final`?

**Comment by rohanKanojia** [comment]
I can confirm that the above test works with the fix cherry-picked on [5.4.x branch](https://github.com/fabric8io/kubernetes-client/tree/5.4.x). We'll do a patch release very soon.

**Comment by rohanKanojia** [comment]
@vsevel : Fix for this issue is available in v5.4.1 . Could you please confirm if v5.4.1 is working for you. If yes, Shall we close this issue?

**Comment by vsevel** [comment]
hello @rohanKanojia 
yes. I had given some feedback in the [quarkus dev gourp](https://groups.google.com/g/quarkus-dev/c/18Ocz1wtKjo)
thanks for your reactivity!
